releasenotes/notes/add-mcx-plugins-85e5b248692a36db.yaml

```diff
@@ -0,0 +1,38 @@
+---
+features_synthesis:
+  - |
+    Added synthesis functions :func:`.synth_mcx_gray_code` and :func:`.synth_mcx_v24`
+    that synthesize multi-controlled X gates. These functions do not require additional
+    ancilla qubits.
+  - |
+    Added synthesis functions :func:`.synth_c3x` and  :func:`.synth_c4x` that
+    synthesize 3-controlled and 4-controlled X-gates respectively.
+features_transpiler:
+  - |
+    Added multiple high-level-synthesis plugins for synthesizing an :class:`.MCXGate`:
```

Comment:
Would you like to add an example code snippet on how this can be used with a special synthesis plugin? That might be cool to see for users 🙂

Reply:
done in [5a8ba64](https://github.com/Qiskit/qiskit/pull/12961/commits/5a8ba642a7024566ef8d78d06a2dd49526d66662)